tests/test_compressors/test_model_compressor.py

```diff
@@ -16,11 +16,24 @@
 
 import pytest
 from compressed_tensors.compressors.model_compressor import ModelCompressor
+from compressed_tensors.config.base import SparsityCompressionConfig
+from compressed_tensors.quantization.quant_config import QuantizationConfig
+
+
+def compressed_tensors_config_available():
+    try:
+        from transformers.utils.quantization_config import (  # noqa: F401
+            CompressedTensorsConfig,
+        )
+
+        return True
+    except ImportError:
+        return False
 
 
 def sparsity_config():
     return {
-        "format": "dense",
+        "format": "sparse-bitmask",  # dense format is ignored by ModelCompressor
```

Comment:
is this change expected?

Reply:
The sparsity config parsing can't be fully tested in the dense case

Reply:
We could test the dense case but it would just be a bunch of Nones